v2.0/app/config/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "IS_LOCAL": false,
+  "IS_LOCAL": true,
```

Comment:
are u sure you wanna deploy with local=true?

Reply:
Done!